modules/label/parser.go

```diff
@@ -36,22 +37,25 @@ func (err ErrTemplateLoad) Error() string {
 // GetTemplateFile loads the label template file by given name,
 // then parses and returns a list of name-color pairs and optionally description.
 func GetTemplateFile(name string) ([]*Label, error) {
-	data, err := options.Labels(name + ".yaml")
-	if err == nil && len(data) > 0 {
-		return parseYamlFormat(name+".yaml", data)
-	}
-
-	data, err = options.Labels(name + ".yml")
-	if err == nil && len(data) > 0 {
-		return parseYamlFormat(name+".yml", data)
-	}
-
-	data, err = options.Labels(name)
-	if err != nil {
-		return nil, ErrTemplateLoad{name, fmt.Errorf("GetRepoInitFile: %w", err)}
+	if filepath.Ext(name) == ".yaml" {
+		data, err := options.Labels(name + ".yaml")
+		if err == nil && len(data) > 0 {
+			return parseYamlFormat(name+".yaml", data)
+		}
+	} else if filepath.Ext(name) == ".yml" {
+		data, err = options.Labels(name + ".yml")
+		if err == nil && len(data) > 0 {
+			return parseYamlFormat(name+".yml", data)
+		}
+	} else {
+		data, err = options.Labels(name)
+		if err != nil {
+			return nil, ErrTemplateLoad{name, fmt.Errorf("GetRepoInitFile: %w", err)}
+		}	
+		return parseLegacyFormat(name, data)
```

Comment:
No need for the else

```suggestion
	}

	data, err = options.Labels(name)
	if err != nil {
		return nil, ErrTemplateLoad{name, fmt.Errorf("GetRepoInitFile: %w", err)}
	}	
	return parseLegacyFormat(name, data)
```